codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -91,6 +119,7 @@ public HttpContentCompressor(int compressionLevel, int windowBits, int memLevel)
         this.compressionLevel = compressionLevel;
         this.windowBits = windowBits;
         this.memLevel = memLevel;
+        this.compressThreshold = (compressThreshold < 0) ? 0 : compressThreshold;
```

Comment:
I'd feel better with throwing if negative.

Reply:
Yes, this is the same as other fields.
@normanmaurer What's your opinion?